lib/apiClient.ts

```diff
@@ -22,5 +28,10 @@ export class ApiClient {
       privilegeLevel: targetPrivilegeLevel,
     });
   }
+
+  public async getPrograms(): Promise<Programs[]> {
+    return (await this.get('/api/programs')) as Programs[];
```

Comment:
🔧 Unsafe casting is scary, you have no idea if this is a `Programs[]` at runtime
Either:
1. Explain in a comment why this assumption is logically sound
2. Enforce a runtime check via Zod